lib/bundler/index.js

```diff
@@ -0,0 +1,132 @@
+// Avoid Browserifying these dependencies
+var crypto, fs, path, through;
+if (module.require) {
+  crypto = require('crypto');
+  fs = module.require('fs');
+  path = require('path');
+  through = require('through');
+}
+
+
+module.exports = function derbyBundler(app, options) {
+  if (process.title === 'browser') {
+    // not in server
+    return;
+  }
+  const { App, util } = app.derby;
```

Comment:
I see a few other uses of `const` and let` in the file, to switch back to `var` for now.

Reply:
fixed them all